tests/baselines/reference/user/acorn.log

```diff
@@ -55,6 +55,7 @@ node_modules/acorn/dist/acorn.es.js(5292,12): error TS2339: Property 'parseExpre
 node_modules/acorn/dist/acorn.js(3,9): error TS2304: Cannot find name 'define'.
 node_modules/acorn/dist/acorn.js(3,34): error TS2304: Cannot find name 'define'.
 node_modules/acorn/dist/acorn.js(3,47): error TS2304: Cannot find name 'define'.
+node_modules/acorn/dist/acorn.js(4,19): error TS2339: Property 'acorn' does not exist on type 'typeof import("/acorn/node_modules/acorn/dist/acorn")'.
```

Comment:
This is a new error from an AMD module which we misread as a commonjs module. Specifically, `global.acorn = global.acorn = {}` no longer has `global: any`.